docs/docs/glossary/react.md

```diff
@@ -3,24 +3,24 @@ title: React
+Facebook lançou o React em 2013. A empresa ainda mantém o projeto, junto com uma comunidade de contribuidores. React é livre para utilizar e de código aberto sob os termos da [licença do MIT](https://github.com/facebook/react/blob/master/LICENSE).
 
-Where publishing tools such as WordPress and Jekyll rely on a system of template files to create a UI, React uses [components](/docs/glossary#component). Components are contained chunks of JavaScript, CSS, and HTML or SVG that can be reused, shared, and combined to create a web site or application.
+Enquanto ferramentas de publicação como WordPress e Jekyll contam com um sistema de arquivos de template para criar uma interface de usuário, React utiliza de [componentes](/docs/glossary#component). Componentes são partes contidas de JavaScript, CSS e HTML ou SVG que pode ser reutilizado, compartilhado e combinado para criar uma página web ou aplicação.
 
-Components may be purely presentational. For example, you might create a `Logo` component that's just an SVG image. Or a component may encapsulate functionality. An `InputBox` component might include an input control, a label, and some simple validation.
+Componentes podem ser puramente de apresentação. Por exemplo, você pode criar um componente `Logo` que é apenas uma imagem SVG. Ou um componente pode encapsular funcionalidades. Um componente `InputBox` pode incluir controle de entrada, um rótulo e uma validação simples.
 
-Components are also _composable_, which is a fancy way of saying that you can use multiple child components to create a parent component or view. This is how you will build Gatsby pages and templates.
+Componentes também são compostáveis, que é um jeito elegante de se dizer que você pode utilizar vários componentes filhos para criar um componente pai ou uma view. É dessa forma que você vai desenvolver páginas e templates Gatsby.
 
-React components respond to changes in _state_. In React, _state_ is a set of properties and values that determine how a component looks or behaves. State can change in response to user activity, such as a click or key press. State can also change as the result of a completed network request. When a value in a component's state changes, the component is the only part of the UI that changes. In other words, React can update part of a page or an entire view without requiring a full page reload.
+Componentes React respondem por mudanças no estado. Em React, estado é um conjunto de propriedades e valores que determina como um componente é apresentado ou se comporta. Estado pode mudar em resposta de uma ação do usuário, como por exemplo um clique do mouse ou uma tecla do teclado. Estado também pode mudar devido a uma requisição de internet. Quando um valor do estado do componente muda, o componente é a unica parte da interface que é alterada. Em outras palavras, React pode atualizar uma parte da página ou uma view inteira sem precisar de um recarregamento completo da página.
```

Comment:
```suggestion
Componentes React respondem por mudanças no estado. Em React, estado é um conjunto de propriedades e valores que determina como um componente é apresentado ou se comporta. O estado pode mudar em resposta de uma ação do usuário, como por exemplo um clique do mouse ou pressionamento de uma tecla. O estado também pode mudar devido a uma requisição de internet. Quando um valor do estado de um componente é alterado, o componente é a única parte da interface que é alterada. Em outras palavras, React pode atualizar uma parte da página ou uma view inteira sem precisar de um recarregamento completo da página.
```

Reply:
Notei que que tem lugares que vc nao usa o artigo definido antes da palavra estado e outros que usa. por exemplo quando vc usa a expressao "do estado". Do = "De + o". O ideal seria manter um padrão, sugiro colocar o artigo em todos os casos pois deixa a leitura mais natural.

Reply:
Para termo em  lingua extrangeira ("view") usar itálico